charts/ui-plugin-server/Chart.yaml

```diff
@@ -1,18 +1,21 @@
 annotations:
   catalog.cattle.io/certified: rancher # Any application we are adding as a helm chart
+  catalog.cattle.io/kube-version: '>= 1.16.0-0 < 1.25.0-0'
```

Comment:
I moved the charts/ contents to charts/ui-plugin-server primarily for semantic reasons. @eliyamlevy do you think we should rename this directory to `chart/` or keep it as `charts/ui-plugin-server`?

Reply:
we can do the rename.